dragonfly/implementations/roach_daq_run_interface.py

```diff
@@ -195,7 +197,7 @@ def _start_data_taking(self, directory, filename):
             logger.error('Local error')
```

Comment:
More details?

Reply:
the error is re-raised after the channel was unblocked. this would be an unexpected error and I don't know more details about it. I dont think this exception has ever been caught so far

Reply:
you can `print(e.message)` to print the message content of the exception or print `('exception while <whatever you were doing>, trying to cleanup')`.

Reply:
@laroque: is there a reason to print the message instead of logger.info it?